src/recipes/ios-promo/config.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
     all: {
       sample: 10 * percent, // will be 1.0 when we launch
       restdays: null,  // irrelevant
-      locales: ["en-US", "en-GB"], // will be en-* locales when we launch
+      locales: ["en-US", "en-GB", "en-ZA"] // All English locales that we support
```

Comment:
en-ZA?  Truly!?  That's neat.  

As an alt solution, consider "en-*" and a regex test.